### ISSUE TITLE
[webpack] debugging problems

### ISSUE THREAD

**Issue by svenefftinge** [theia-ide/theia#302]
Somehow there are multiple copies of a module loaded (they get different suffixes). E.g.:

<img width="558" alt="screen shot 2017-07-15 at 10 00 23" src="https://user-images.githubusercontent.com/372735/28237641-7edf7338-6944-11e7-8e77-b9440517580c.png">

If you want to debug by setting a breakpoint you need to set it in the right file, to have it hit.

@akosyakov could that be related to the recent change in our webpack config where it now builds a chink per extension?

**Comment by akosyakov** [comment]
yes, you are right, i will remove code splitting, it introduces a lot of duplications in such way

**Comment by akosyakov** [comment]
Fixed with #319
